README.md

```diff
@@ -16,7 +16,23 @@ bitbucketApi.user.get().then(({ body }) => {
 
 ### `options`
 It is not necessary to provide any options at all (`Bitbucket` can be constructed with no argument).
- - `useXhr` (`Boolean`): If `true`, requests will be made using XMLHttpRequest. This is only available in a web browser, and will fail otherwise. This can be very useful in Electron for automatically resolving proxies and custom SSL certificates.
- - `proxy` (`String`): Defines a proxy to make requests against, instead of `api.bitbucket.org:443`. This option is _ignored_ when `useXhr` is active.
+ - `requesterFn` (`(options) => Promise<any>`): If provided, requests will be made using the function you provide. This is allows you to use your preferred http client. The `options` provided are `{ headers, hostname, method, path, query, url, body? }`. `body` is only provided on `POST` methods. In the case of `doPrebuiltSend` (e.g. pagination), only `{ headers, method, url }` are provided in the options. Example:
```

Comment:
Nobody knows what doPrebuiltSend is. The goal here is to prevent people from having to read the source code.

Let's walk backwards from doPrebuiltSend: It's used in getNextPage, getPreviousPage, getForksFromResponse and getParentFromResponse. These are functions that take a response as their argument; so that's a good way to describe when options will be missing. True, the methods aren't documented yet, but if and when they are, this section will already be accurate.